engines/router/missionctl/fiberapi/routing_strategy.go

```diff
@@ -64,23 +64,20 @@ func (r *DefaultTuringRoutingStrategy) SelectRoute(
 		close(expResultCh)
 	}
 
-	// If error, return it
+	// If error, log it and return the fallback(s)
 	if expErr != nil {
 		log.WithContext(ctx).Errorf(expErr.Error())
-		return nil, fallbacks, createFiberError(expErr)
+		return nil, fallbacks, nil
```

Comment:
Considering this change, I don't think we're returning any errors anymore? So perhaps the 3rd response object could be removed.

Reply:
Good question. This method implements an interface exposed by Fiber ([here](https://github.com/gojek/fiber/blob/4f0f8284dc84cdab289715eecd70b88213e5133c/routing_strategy.go#L11)). We could certainly consider removing it but we have to examine the usage more closely before making that change (i.e., even if it has 0 benefits within the Turing router, should Fiber, as a standalone library, stop supporting it?). I think we will cover this when we get to the error handling improvements in Turing.